apps/mobile-wallet/src/screens/Dashboard/useLoader.ts

```diff
@@ -36,13 +36,28 @@ export const useLoader = () => {
   const [showLoader, setShowLoader] = useState(addressesStatus === 'uninitialized')
 
   useEffect(() => {
-    if (progress === 1) {
-      setTimeout(() => setShowLoader(false), 500)
+    let timeoutID: NodeJS.Timeout
+
+    if (progress >= 1) {
+      timeoutID = setTimeout(() => setShowLoader(false), 300)
     }
+
+    return () => clearTimeout(timeoutID)
```

Comment:
@mvaivre This can be done simpler. I'll fix it

```ts
    if (progress >= 1) {
      const timeoutId = setTimeout(() => setShowLoader(false), 300)

      return () => clearTimeout(timeoutId)
    }
```

Reply:
Done!